src/ui/server/controllers/insurance/policy-and-export/type-of-policy/index.test.ts

```diff
@@ -204,6 +206,7 @@ describe('controllers/insurance/policy-and-export/type-of-policy', () => {
     describe('api error handling', () => {
       beforeEach(() => {
         req.body = {
+          // @ts-ignore
```

Comment:
Why do we have the ts-ignores?

Reply:
With the things I have added it was causing a linting error in the tests and could not figure out why